src/renderers/dom/shared/__tests__/ReactDOMServerIntegration-test.js

```diff
@@ -298,6 +330,7 @@ function resetModules() {
   // TODO: can we express this test with only public API?
   ExecutionEnvironment = require('ExecutionEnvironment');
 
+  require('ReactFeatureFlags').disableNewFiberFeatures = false;
```

Comment:
New features are now enabled for all fixtures in this file. We don't ship "no new features" SSR builds anymore.